components/TripList/index.js

```diff
@@ -8,11 +8,15 @@ export default function TripList() {
     return <h2>... is loading</h2>;
   }
 
+  const sortedtrips = trips
```

Comment:
Same as above, please write variable names in camelCase.

```suggestion
  const sortedTrips = trips
```